text/0001-first-version-architecture.md

```diff
@@ -0,0 +1,85 @@
+- Start Date: 19/06/2018
+- RFC PR:
+
+
+# Summary
+
+Here we will discuss the architecture for the first version of the Enactus Team Management System.
+This involves which features will be developed, which classes/models
+will be added and how they relate to each other.
+
+# Motivation
+
+We want the Enactus Team Management System to be open source and an opportunity of learning
+to any Enactus developer. The first opportunity we see is to provide our first developers
+the chance to elaborate a good and well thought architecture for the first version of the software
+seeking a great code quality application.
+
+
+# Detailed design
+
+## Functional Requirements
+
+### Signup a team
+A user must be able to signup his team by providing these informations
+* `email`
+* `university / institution`
+* `enactus team ID`
+* `name`
+
+### Invite team members
+After a team is successfully signed up, an e-mail must be sent to
+the team's email containing a link to invite members.
+
+On signup, each member must provide the following information
+
+* `email`
+* `password`
+* `name`
+* `documents` (`CPF` is required for certificates, `RG` is optional)
```

Comment:
Must create a `documents` table, with attributes `documentType` and `documentID`.

Each user must have multiple documents. When signin up, a user must provide at least one document.